lux-lib/src/lockfile/mod.rs

```diff
@@ -1336,7 +1437,7 @@ mod tests {
 
     fn get_test_lockfile() -> Lockfile<ReadOnly> {
         let sample_tree = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
-            .join("resources/test/sample-tree/5.1/lock.json");
+            .join("resources/test/sample-tree/5.1/lux.lock");
```

Comment:
todo:

- [ ] update/rename snapshot files (I think this is why the tests are failing).